Sources/CredentialsGoogle/CredentialsGoogleToken.swift

```diff
@@ -77,10 +80,17 @@ public class CredentialsGoogleToken: CredentialsPluginProtocol {
                 #else
                     let key = token as NSString
                 #endif
-                let cacheElement = usersCache!.object(forKey: key)
-                if let cached = cacheElement {
-                    onSuccess(cached.userProfile)
-                    return
+                if let cached = usersCache?.object(forKey: key) {
+                    if let ttl = tokenTimeToLive {
+                        if Date() < cached.createdAt.addingTimeInterval(ttl) {
+                            onSuccess(cached.userProfile)
+                            return
+                        }
```

Comment:
Could we add a comment explaining that if the token is expired, it doesn't need explicitly evicting as we will replace it by going through the normal auth flow and creating a new UserProfile to store in the cache.